tests/scenario/migrated/model-controller.test.ts

```diff
@@ -127,6 +127,23 @@ describe("features/Model/Controller", () => {
     await expect(promise).rejects.toMatchObject({ status: 404 });
   });
 
+  it("List on an engine is also returning commons assets models", async () => {
+    const response = await sdk.query({
+      controller: "device-manager/models",
+      action: "listAssets",
+      engineGroup: "engine-ayse",
+    });
+
+    expect(response.result).toMatchObject({
+      total: 3,
+      models: [
+        { _id: "model-asset-Container" },
+        { _id: "model-asset-Plane" },
+        { _id: "model-asset-Warehouse" },
+      ],
+    });
+  });
```

Comment:
IMHO this test could be improved
Your test case check the same thing of the previous test (`Write and List an Asset model`).
```js
    response = await sdk.query({
      controller: "device-manager/models",
      action: "listAssets",
      engineGroup: "commons",
    });
```

So that doesn't really test if commons templates are included with engine models or if it's return instead of.
Could you add a specific engine models to ensure, that is correctly returned with commons models ?

Reply:
All models are commons models but yeah not perfect maybe I should declare a new model on an engine to. 

List is used with common in the other test and with an engine there

Reply:
done, use other-group in place of ayse that is in commons group